pkg/threescale/backend/manager.go

```diff
@@ -0,0 +1,367 @@
+package backend
+
+import (
+	"fmt"
+
+	"github.com/3scale/3scale-go-client/threescale/api"
+
+	"github.com/3scale/3scale-go-client/threescale"
+)
+
+// Manager manages connections to multiple backends
+type Manager struct {
+	connections map[string]Backend
```

Comment:
We mean to use URLs for the 3scale backend here (or any other unique id), don't we?

Reply:
Yes, URLs